### ISSUE TITLE
tweak error message format

### ISSUE THREAD

**Issue by zargot** [Big-Life-Lab/PHES-ODM-Validation#23]
Just a small issue. The convention for the ODM is `[table]_[part]` not  `[table].[part]`. We do that because using `_` generates a valid name in all programming languages. (`.`) does not.

_Originally posted by @DougManuel in https://github.com/Big-Life-Lab/PHES-ODM-Validation/issues/21#issuecomment-1236329240_

**Comment by zargot** [comment]
I'm assuming this is about the error message format. I'll see if I can make it more proper. Maybe I can add another rule before this, with a different kind of error message, so that I can get a better idea of how to structure the code?

**Comment by DougManuel** [comment]
Yes. The message could read more in a human-readable format for the error report. Error found on table [table] in part [part].

The format of combining tables and variables in how to name parts in "wide" table format, which doesn’t apply to the validation tools at this point. We are only validating 'long' tables.
